xclim/indices/_threshold.py

```diff
@@ -649,6 +650,46 @@ def maximum_consecutive_dry_days(pr, thresh="1 mm/day", freq="YS"):
     return group.apply(rl.longest_run, dim="time")
 
 
+@declare_units("days", tasmax="[temperature]", thresh="[temperature]")
+def maximum_consecutive_summer_days(tasmax, thresh="25 degC", freq="YS"):
```

Comment:
Name change?

Reply:
Idem. no change to this in this branch. sorry about the confusion.